eureka-core/src/main/java/com/netflix/eureka/resources/InstanceResource.java

```diff
@@ -118,7 +118,7 @@ public Response renewLease(
         }
         // Check if we need to sync based on dirty time stamp, the client
         // instance might have changed some value
-        Response response = null;
```

Comment:
This change is unrelated and I'd suggest to submit a separate PR for style only fixes if you're interested.